src/components/preview-icon/PreviewIcon.tsx

```diff
@@ -0,0 +1,15 @@
+import React from "react";
+import { SPreviewIcon } from "./styles/SPreviewIcon";
+
+export interface IPreviewIcon {
+  icon?: React.ReactNode;
+  size: number;
+}
+
+export const PreviewIcon: React.FC<IPreviewIcon> = ({ icon, ...props }) => {
```

Comment:
This component is repeated and not actually needed, let's discuss on why the original is not being used.